portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,23 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <ul id = "navbar">
+      <li id = "nav"><a href="about.html" style="text-decoration: none;">about</a></li>
+      <li id = "nav"><a href="/documents/resume.pdf" style="text-decoration: none;">resume</a></li>
+      <li id = "nav"><a href="work.html" style="text-decoration: none;">work</a></li>
+      <li id = "nav"><a href="mailto:bsaputra132@gmail.com" style="text-decoration: none;">contact</a></li>
+    </ul>  
+
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Hi, I'm Bianca.</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <p>I'm a junior at Duke University majoring computer science and minoring in computational biology and religion.</p>
+      <img src = "/images/duke.jpg" alt="Duke Chapel" />
+      <p></p>
       <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="greeting-co
+      ntainer"></div>
```

Comment:
why is this separated onto different lines?

Reply:
Fixed!